lib/config/folderconfiguration.go

```diff
@@ -43,17 +44,23 @@ func (f FolderConfiguration) Copy() FolderConfiguration {
 }
 
 func (f FolderConfiguration) Filesystem() fs.Filesystem {
```

Comment:
Is this one really used? It seems possibly error prone to keep around, maybe if it's only used from tests and stuff we could make this the main thing and have callers do `Filesystem(nil)` when they specifically want a non-mtime version of it?

Reply:
It is used in quite a few places. Mostly for "simple operations" where mtime-handling doesn't matter (.stfolder, getting the filesystem uri, ...). Just the file versioner uses mod-time, but is not an mtime-filesystem. That's affected by the same issues around mtime as the "main folder filesystem", so likely it would be good to have it there too - haven't thought it trough.